stable/nuclio/templates/_helpers.tpl

```diff
@@ -0,0 +1,23 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "nuclio.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "nuclio.controllerName" -}}
+{{- $name := default .Chart.Name .Values.Controller.Name -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "nuclio.controllerImage" -}}
```

Comment:
I don't like these templates for image. Please check our best practices in the review guidelines.

Reply:
There's nothing against these templates in the guidelines. The way nuclio are tagging their images - it will easier to template them (and isn't that the purpose of `_helpers.tpl` file?)

Reply:
It is non-standard and and unnecessary indirection because it is only used in one place. From a user's perspective it makes things more complicated.